textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -323,26 +324,40 @@ public static String getSpanString(Span span) {
         return sb.toString();
     }
 
-    public static List<ITuple> removePayload(List<ITuple> tupleList) {
-        List<ITuple> tupleListWithoutPayload = tupleList.stream().map(tuple -> removePayload(tuple))
+    /**
+     * Remove one or more fields from each tuple in tupleList.
+     * 
+     * @param tupleList
+     * @param removeFields
+     * @return
+     */
+    public static List<ITuple> removeField(List<ITuple> tupleList, String... removeFields) {

```

Comment:
Should we rename `removeField` to `removeFields` since it can have more than one field?